WordPress/Classes/Utility/Media/ImageDownloader.swift

```diff
@@ -2,6 +2,8 @@ import UIKit
 
 struct ImageRequestOptions {
     /// Resize the thumbnail to the given size. By default, `nil`.
+    ///
+    /// - warning: The size is in pixels.
     var size: CGSize?
```

Comment:
Should probably add something like `typealias ImageSize = CGSize` in the future to avoid having to document it everywhere. There is an existing enum `ImageSize`, but it doesn't work the way I need it to.